### ISSUE TITLE
Versioncheck reporting wrong version

### ISSUE THREAD

**Issue by jantman** [jantman/awslimitchecker#55]
Using python virtualenv==1.11.4 and Python 2.7.10, when awslimitchecker is installed as a pip requirement of another package, the version information is incorrectly output:

```
WARNING:awslimitchecker.checker:awslimitchecker ('00000000', '00000001', '*final') is AGPL-licensed free software; all users have a right to the full source code of this version. See <https://github.com/jantman/awslimitchecker>
```

It looks like we shouldn't be using parsed_version.

**Comment by jantman** [comment]
This issue is caused by using pip < 6.0, where parsed_version is a tuple instead of a Version object.

**Comment by jantman** [comment]
This has been released in 0.1.1, which is now live on pypi.
